src/dials/util/system.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
```

Comment:
Not reviewing these changes as assume they were pulled verbatim from the upstream source. 
I trust that the proposal as made is a sensible one.

Reply:
OK.